__app__/adparser/sites/sumosear_ch.py

```diff
@@ -0,0 +1,86 @@
+from typing import List
+from __app__.adparser.sites.base_ad_parser import BaseAdParser
+import re
+
+class SumoSear_ch(BaseAdParser):
+    def primary_phone_number(self) -> str:
+        number = self.soup.find("a", class_="card-info__tel-num")
+        if number:
+            number = number.get_text()
+        return number
```

Comment:
```suggestion
        if number:
            return number.get_text()
```
If number is None, we don't need to return it. Python automatically returns None if we don't return something at the end of the function.